test/test_autograd.py

```diff
@@ -7562,6 +7562,18 @@ def test_disabling_saved_tensor_hooks_nested(self):
 
         self.assertTrue(torch._C._autograd._saved_tensors_hooks_is_enabled())
 
+    def test_ignore_disable_saved_tensor_hooks(self):
+        with torch.autograd.graph._ignore_disable_saved_tensor_hooks():
+            with torch.autograd.graph.disable_saved_tensors_hooks("error message"):
+                self.assertTrue(torch._C._autograd._saved_tensors_hooks_is_enabled())
+                with torch.autograd.graph.saved_tensors_hooks(lambda x: x, lambda x: x):
+                    pass
```

Comment:
If this does nothing it is not really testing anything right?

Reply:
We don't need to actually trigger the hook, simply entering the context manager would trigger the error.